samples/geography/requirements.txt

```diff
@@ -24,14 +24,12 @@ importlib-metadata==4.8.1
 libcst==0.3.21
 munch==2.5.0
 mypy-extensions==0.4.3
-numpy==1.19.5; python_version < "3.7"
-numpy==1.21.2; python_version > "3.6"
```

Comment:
Was `numpy` a redundant dependency?

Reply:
Yes. We don't use it directly, it comes in as a dependency of pandas. Rather than try and find the right combo of versions to support Python 3.10, I just dropped it.